.github/workflows/e2e.yml

```diff
@@ -32,12 +32,23 @@ jobs:
           key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
           restore-keys: |
             ${{ runner.os }}-yarn-
+
+      - name: create cache folder for Cypress
+        id: cypress-cache-dir-path
+        run: echo "::set-output name=dir::$(mktemp -d)"
+      - uses: actions/cache@v3
+        with:
+          path: ${{ steps.cypress-cache-dir-path.outputs.dir }}
+          key: ${{ runner.os }}-cypress
       - name: Install Node.js
         uses: actions/setup-node@v3
         with:
           node-version: lts/*
       - name: Install dependencies
         run: corepack yarn install --immutable
+        env:
+          # https://docs.cypress.io/guides/references/advanced-installation#Binary-cache
+          CYPRESS_CACHE_FOLDER: ${{ steps.cypress-cache-dir-path.outputs.dir }}
```

Comment:
I wonder how this can work without centralized remote caching? We get a random new machine on GitHub Actions every time AFAIK so this would never be present?

Reply:
See https://github.com/actions/cache for more info.